### ISSUE TITLE
better describe the use of decimal-separated value for `group-position`

### ISSUE THREAD

**Issue by rdeltour** [w3c/epub-specs#1128]
The `meta` property `group-position` can have a value of type integer or decimal-separated-value (e.g. 2.2.1). What does the latter means? How is it used to "indicate the numeric position in which the EPUB Publication is ordered relative to other works belonging to the same group"?

If we allow complex schemes like decimal-separated values, shouldn't the entire value space be open-ended to allow other value schemes? Is it an arbitrary decision or a common practice?

Just wonderin' 😄

**Comment by mattgarrish** [comment]
As far as memory serves, it was only supposed to take an integer originally. Adding decimal-separated values for more complex cases was a compromise that would still enable machine sorting without rendering the field completely useless for any kind of sorting purposes.

**Comment by dauwhe** [comment]
@rdeltour perhaps we could add an example of a decimal-separated `group-position`? This is Volume 98, Issue 4 of Physical Review D: 

```xml
<meta property="belongs-to-collection" id="cygnus-x-1">Physical Review D</meta>
<meta refines="#cygnus-x-1" property="collection-type">series</meta>
<meta refines="#cygnus-x-1" property="group-position">98.4</meta>
```

**Comment by dauwhe** [comment]
Closed via #1140
